src/layouts/MapLayout.tsx

```diff
@@ -85,6 +110,9 @@ export const MapLayout: FC<MapLayoutType> = ({
             zoom: zoom || MAP_CONFIG.defaultZoom,
           }}
           onSelect={onTreeSelect}
+          onOutdatedNowcastCheck={(isOutdated) =>
```

Comment:
We could maybe rename this function to `onOutdatedNowcastChange` because `check` isn't very telling in my opinion. Feel free to ignore though.

Reply:
Goo idea. This specific line doesn't exist anymore, but I've changed the name of the function in other places.